lib/main_common.pm

```diff
@@ -1101,12 +1101,13 @@ sub load_inst_tests {
         loadtest "installation/installation_overview";
         # On Xen PV we don't have GRUB on VNC
         # SELinux relabel reboots, so grub needs to timeout
-        set_var('KEEP_GRUB_TIMEOUT', 1) if check_var('VIRSH_VMM_TYPE', 'linux') || get_var('SELINUX');
+        set_var('KEEP_GRUB_TIMEOUT', 1) if check_var('VIRSH_VMM_TYPE', 'linux') || (get_var('SELINUX') && is_sle('<16'));
         loadtest "installation/disable_grub_timeout" unless get_var('KEEP_GRUB_TIMEOUT');
         if (check_var('VIDEOMODE', 'text') && is_ipmi) {
             loadtest "installation/disable_grub_graphics";
         }
-        loadtest "installation/enable_selinux" if get_var('SELINUX');
+        # Do not run enable_selinux in tumbleweed as it is selected as default (bsc#1230118)
```

Comment:
IMO instead of `is_tumbleweed` this should get a new `has_selinux_by_default` method in `version_utils`. That way the check is in a single place.

Reply:
ftr, i still need to test the change